README.md

```diff
@@ -90,457 +90,467 @@ type: tutorial
-    storageclass_name: <existing_storageclass>
-    storage_size: 20G
-```
+    ```yaml
+    apiVersion: labs.ai/v1alpha1
+    kind: Eddi
+    metadata:
+      name: eddi
+    spec:
+      size: 1
+      mongodb:
+        environment: prod
+        storageclass_name: <existing_storageclass>
+        storage_size: 20G
+    ```
 
-![ocp-createyaml](doc/source/images/ocpcreateyml.png)
+    ![ocp-createyaml](images/ocpcreateyml.png)
 
-- E.D.D.I Operator pods should come up when the installation is completed.
+    E.D.D.I Operator pods should come up when the installation is completed.
 
-- Under the left panel click on `Networking` and select `Routes`, make sure you are in the `eddi-project` namesapace, you can now see the `eddi-route` with a URL as shown.
+1. Under the left panel, select **Networking > Routes**. Make sure you are in the `eddi-project` namesapace. You should now see the `eddi-route` with a URL like the following: 
 
-![ocp-routes](doc/source/images/ocproutes.png)
+    ![ocp-routes](images/ocproutes.png)
 
-- You can now visit the URL to access the E.D.D.I Dashboard.
+1. You can now visit the URL to access the E.D.D.I Dashboard.
 
-![ocp-eddidashboard](doc/source/images/eddidashboard.png)
+    ![ocp-eddidashboard](images/eddidashboard.png)
 
 ### Step 4: Create a chatbot in E.D.D.I
 
-- Now, let's create a chatbot in E.D.D.I from scratch.
+Now, let's create a chatbot in E.D.D.I from scratch.
+
+To create a chatbot, you will complete these steps: <!--EM: I feel like, at the very least, we need to link people to where they can learn more about these concepts (regular dictionary, behavior rules, output sets). Is this common vocab for developing chatbots? Why would readers need to develop these things? This feels so advanced and I'm worried about just dropping our readers into this without more explanatory text -->
 
-- The steps involved in creating a chatbot are as follows:
-    - Create a `Regular Dictionary`
-    - Create `Behavior Rules`
-    - Create `Output sets`
-    - `Packaging` it
-    - Create a `bot`
+1. Create a regular dictionary
+1. Create behavior rules
+1. Create output sets
+1. Package your dictionary, rules, and sets
+1. Create a chatbot
 
-- In order to build a Chatbot with E.D.D.I, you will have to create a few configuration files and POST them to the corresponding REST APIs. You can use tools like [Postman](https://www.postman.com/downloads/) to make the API calls.
+In order to build a Chatbot with E.D.D.I, you will have to create a few configuration files and POST them to the corresponding REST APIs. You can use tools like [Postman](https://www.postman.com/downloads/) to make the API calls.
 
-#### Step 4.1: Create Regular Dictionary
+#### Step 4.1: Create a regular dictionary
 
-- We create regular dictionaries in order to store custom words and phrases.
+*Regular dictionaries* are used store custom words and phrases. <!--EM: We should clarify this. Is that in machine learning world? AI world?->
```

Comment:
**Is that in machine learning world? AI world?**
Didn't really understand your question but, AI is an umbrella term for Machine Learning. Machine Learning is a part of AI. We can use AI world, it gives broader perspective .